vars/withZalenium.groovy

```diff
@@ -1,148 +0,0 @@
-/**
```

Comment:
Hmm the point of the zalenium build lib is to use Zalenium. I see what you doing here as Zalenium is out-of-support. 

Anyhow I suggest to do a breaking build and add properly documented what happens here as well as an out-of-maintenance or something similar.